metrics/ci_integrations/test/source/github_actions/config/validator/github_actions_source_validator_test.dart

```diff
@@ -0,0 +1,517 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:ci_integration/client/github_actions/models/github_token.dart';
+import 'package:ci_integration/client/github_actions/models/github_token_scope.dart';
+import 'package:ci_integration/integration/validation/model/field_validation_result.dart';
+import 'package:ci_integration/integration/validation/model/validation_result.dart';
+import 'package:ci_integration/integration/validation/model/validation_result_builder.dart';
+import 'package:ci_integration/source/github_actions/config/model/github_actions_source_config.dart';
+import 'package:ci_integration/source/github_actions/config/model/github_actions_source_config_field.dart';
+import 'package:ci_integration/source/github_actions/config/validation_delegate/github_actions_source_validation_delegate.dart';
+import 'package:ci_integration/source/github_actions/config/validator/github_actions_source_validator.dart';
+import 'package:ci_integration/source/github_actions/strings/github_actions_strings.dart';
+import 'package:ci_integration/util/authorization/authorization.dart';
+import 'package:ci_integration/util/model/interaction_result.dart';
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+
+import '../../../../test_utils/extensions/interaction_result_answer.dart';
+
+void main() {
+  group("GithubActionsSourceValidator", () {
+    const accessToken = 'accessToken';
+    const repositoryOwner = 'repositoryOwner';
+    const repositoryName = 'repositoryName';
+    const workflowId = 'workflowId';
+    const jobName = 'jobName';
+    const coverageArtifactName = 'coverageArtifactName';
+    const message = 'message';
+
+    final config = GithubActionsSourceConfig(
+      accessToken: accessToken,
+      repositoryOwner: repositoryOwner,
+      repositoryName: repositoryName,
+      workflowIdentifier: workflowId,
+      jobName: jobName,
+      coverageArtifactName: coverageArtifactName,
+    );
+
+    final auth = BearerAuthorization(accessToken);
+
+    const githubToken = GithubToken(
```

Comment:
Would it be more readable if we declare the values in the following order: 
1. `const` values
2. `final` values
3. functions

What do you think?